packages/apps/plugins/plugin-help/src/components/Shortcuts/ShortcutsDialog.tsx

```diff
@@ -26,7 +27,6 @@ export const ShortcutsDialogContent = () => {
   const { t } = useTranslation('os');
```

Comment:
ok, so this is good enough to handle all of the plugin-specific translations?

Reply:
ah missed this one, this should be `HELP_PLUGIN`